.vimrc

```diff
@@ -905,7 +936,7 @@
 
                     " <CR>: close popup
                     " <s-CR>: close popup and save indent.
-                    inoremap <expr><s-CR> pumvisible() ? neocomplcache#close_popup()."\<CR>" : "\<CR>"
+                    inoremap <expr><s-CR> pumvisible() ? neocompl_`#close_popup()."\<CR>" : "\<CR>"
```

Comment:
That was not. I will fix it. I was eager to stop working and open a PR ASAP as this PR was already huge